src/poliastro/earth/sensors.py

```diff
@@ -95,6 +93,7 @@ def max_and_min_ground_range_with_specific_azimuth(
     -----
     For further information, please take a look at "Fundamentals of Astrodynamics and Applications", 4th ed (2013)"
     by David A. Vallado, pages 853-860.
+
```

Comment:
No need for this ... I have tested this change ... not required IMO

Reply:
This was done to preserve uniformity across all docstrings...So I think it is required..